.github/workflows/test-paradedb.yml

```diff
@@ -58,6 +58,9 @@ jobs:
           endpoint: ${{ secrets.DOCKERHUB_USERNAME }}/paradedb
           install: true
 
+      - name: Install Rust
```

Comment:
@rebasedming This won't fix it, the Rust install inside the Docker image. You need to replace the rust version of `1.80` to `stable` inside the Dockerfile